packages/apollo-mock-client/README.md

```diff
@@ -0,0 +1,258 @@
+# ApolloMockClient
+
+An Apollo Client that allows mocking of payloads in response to operations, rather than having to provide them all upfront. It is API-wise a port of [Relay’s RelayMocEnvironment](https://relay.dev/docs/guides/testing-relay-components/#relaymockenvironment-api-overview).
```

Comment:
```suggestion
An Apollo Client that allows mocking of payloads in response to operations, rather than having to provide them all upfront. It is API-wise a port of [Relay’s RelayMockEnvironment](https://relay.dev/docs/guides/testing-relay-components/#relaymockenvironment-api-overview).
```